### ISSUE TITLE
All data is being deleted on restart

### ISSUE THREAD

**Issue by alchemistgo87** [jhipster/generator-jhipster#2075]
I have generated my jhipster app with mongodb database. There is a strange problem I am facing. When I restart my app, all the data inside mongodb database is deleted.

**Comment by deepu105** [comment]
@alchemistgo87 dude i guess this is not the first time you are posting here, plz read our issue guidelines, we cannot help you with the data you have provided, so im closing this. Plz understand that we are doing this in our free time that means we dont have time to go around investigating an issue without knowing any details of your config, use case etc. I usually wont provide an explanation,  this is just since you seem to be opening issues now and then

**Comment by vasuneeli1** [comment]
I faced the same problem with eclipse Kepler, Java 1.7. 

I downloaded the Jhipster 2 days ago. So I had the latest version. Not sure how to check the JHipster current version though.

I ran the application from eclipse and selected run as Spring boot application. Application was starting without any errors and it functions. But when i restart the server, no data exists.

I couldn't figure out which mongo db it is using, because my application was working even without starting mongo DB.

After struggling for one and half day, I read the instructions again and ran the command from command line.
mvn spring-boot:run

from this document. http://jhipster.github.io/installation.html

Now i see errors and had to start my mongodb manually and see a new connection opened. And created database and all collections.

From web, i found out that it works well with spring tool suite, i downloaded the latest version and ran the same main class and selected run as Spring boot application.

It opened a new connection and working fine. Able to see the data after restarts too.

I am not sure whether it is an issue with a particular version of Eclipse or the combination caused it.

Thanks.
Satya
